data/tlg2001/tlg043/tlg2001.tlg043.opp-lat1.xml

```diff
@@ -4,7 +4,7 @@
   <teiHeader>
     
     <fileDesc>
-  <titleStmt><title>COMMENTARIA IN ARISTOTELEM GRAECA</title>
+      <titleStmt><title xml:lang="lat">COMMENTARIA IN ARISTOTELEM GRAECA</title>
```

Comment:
If there is text in any of the element in the teiHeader that are in all caps, please change these to be only in title case. This includes the `<title>` and the `<editor>` elements here. There are also other elements below that should be corrected in the same way.
Also, add an `<author>` element here with the text "Themistius".
Also, the `<idno type="filename">TO BE ADDED BY LEIPZIG</idno>` should be corrected to show the correct filename for this file.
In the `<listBibl>` element below, the following changes should be made:

- The `<title>COMMENTARIA IN ARISTOTELEM GRAECA</title>` should have an `xml:lang="lat"` attribute (plus it should be made title case instead of all caps):
- The `<name xml:lang="lat">SAMUEL LANDAUER</name>` element should have the `xml:lang="lat"` attribute removed (and it should be made title case).
- The `<author ref="urn:tmp">SAMUEL LANDAUER</author>` element  should be changed to "Themistius" and the `ref` attribute should have the URN for the Themistius text group, i.e., "urn:cts:greekLit:tlg2001".
- The `<publisher>typis et impensis georgii reimeri</publisher>` element should be correctly capitalized and it should have an `xml:lang="lat"` attribute added to it.
- The `<pubPlace>BEROLINI</pubPlace>` element should be in title case and should have an `xml:lang="lat"` attribute added to it.
- The `<date>MCMII</date>` attribute should be written in Arabic numerals instead of Roman, i.e., "1902".
- The `<biblScope unit="volume">vol. v</biblScope>` element should only contain the Arabic numeral for the volume here. So instead of "vol. v", it should be "5".